iamctl/pkg/utils/keywordUtils.go

```diff
@@ -348,3 +349,16 @@ func AddTypeTags(data []byte) []byte {
 	re := regexp.MustCompile(`1typeTag: `)
 	return re.ReplaceAll(data, []byte("!!org.wso2."))
 }
+
+func ReplacePlaceholders(configFile []byte) []byte {
```

Comment:
Are we handling scenarios where an environment variable value is not set for a variable added to the file?

Reply:
Currently, the keyword placeholder will not be replaced by any other value and keep the `${PLACE_HOLDER_NAME}` when importing.